src/commands/commands.ts

```diff
@@ -685,6 +685,163 @@ const debugCmd = new SimpleAction(
   },
 );
 
+/** Label command, used to change the label of the given channel. */
+const labelCmd = new TwoPartCommand(
+  'label',
+  `Change the channel's label for easier debugging.`,
+  'label <bot name> <channel id> <channel label>',
+  // Group trigger
+  /^\s*label(?<group>.*)$/,
+  // Action trigger
+  /^\s*(?:(?<botName>\w+)\s+)?(?<channelID>(?:(?:\+|-)?\d+)|(?:this))\s+(?<channelLabel>.+?)\s*$/,
+  // Action
+  async (message, match) => {
+    const bot = message.getBot();
+    const { botName, channelID, channelLabel } = match.groups;
+
+    let channelBot = bot;
+    // Get the bot to find the channel on
+    if (botName) {
+      const findBot = getBots().find((client) => client.name === botName);
+
+      if (findBot) {
+        channelBot = findBot;
+      } else {
+        // Did not find the specified bot
+        const botList = getBots()
+          .map((client) => `- ${client.name}`)
+          .join('\n');
+
+        message.reply(
+          `'${botName}' is not an available bot! Try one of the following:\n${botList}`,
+        );
+      }
+    } else {
+      // No bot specified, take the bot that the message was written on
+      channelBot = bot;
+    }
+
+    let labelChannel = message.channel;
+
+    if (channelID === 'this') {
+      // Take this channel and this bot
+      labelChannel = message.channel;
```

Comment:
very minor but this line seems unnecessary

Reply:
Removed.